packages/server/graphql/mutations/renameMeetingTemplate.ts

```diff
@@ -33,8 +33,9 @@ const renameMeetingTemplate = {
     if (!template || !template.isActive) {
       return standardError(new Error('Template not found'), {userId: viewerId})
     }
-    if (!isTeamMember(authToken, template.teamId)) {
-      return standardError(new Error('Team not found'), {userId: viewerId})
+    const isAuthorized = await isTeamMemberOrOrgAdmin(authToken, template.teamId, dataLoader)
```

Comment:
(for future reference) This diff has the right intent, but could have been written simpler using the existing `isUserOrgAdmin` function:

https://github.com/ParabolInc/parabol/blob/abce357358353cd4844d99e349177b1bc38ab4fc/packages/server/utils/authorization.ts#L50-L57